tools/simulator/libsimulator/lib/ProjectConfig/ProjectConfig.h

```diff
@@ -45,6 +48,8 @@ class CC_LIBSIM_DLL ProjectConfig
 
     static const int DEFAULT_WIDTH = 640;
     static const int DEFAULT_HEIGHT = 960;
+	static const ResolutionPolicy DEFAULT_RESOLUTION_POLICY = ResolutionPolicy::EXACT_FIT;
+    static constexpr float DEFAULT_CONTENT_SCALE_FACTOR = 1.0f;
```

Comment:
"constexpr" is a C++11 new feature, it can not compiling under VisualStudio 2013, this break cocos2dx compatibility, please fix it.